src/map.cpp

```diff
@@ -4737,6 +4737,11 @@ item map::water_from( const tripoint &p )
     }
 
     const ter_id terrain_id = g->m.ter( p );
+    if( terrain_id == t_sewage ) {
+        item ret( "sewage", 0, item::INFINITE_CHARGES );
```

Comment:
Where do you suggest to place it - `chemicals_and_resourses.json` or `comestibles.json`?